src/components/Modals/AddContactModal.jsx

```diff
@@ -49,28 +55,61 @@ const AddContactModal = ({ addContact, showAddContactModal, setShowAddContactMod
   const [userFamilyName, setUserFamilyName] = useState('');
   const [webId, setWebId] = useState('');
   const [invalidWebId, setInvalidWebId] = useState(false);
+  const [userName, setUserName] = useState('');
+  const [customWebID, setCustomWebID] = useState(false);
   const [processing, setProcessing] = useState(false);
   const theme = useTheme();
   const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
+  const [oidcProviders] = useState([...ENV.VITE_SUGGESTED_OIDC_OPTIONS.split(', '), 'Other']);
+  const [Oidc, setOIDC] = React.useState('');
+
+  const oidcForm = {
```

Comment:
Rather than hardcode these in the component, I think it would make sense to place them in the ENV variable in some way. That way both the options for default OIDC providers, and their web ID templates are in the same place.

Reply:
Makes sense, I had a similar idea. I think I can move them out safely into the env file. I would have to update the test again to include the new env variables but should be easy.

Reply:
Okay I was able to pull out the WebID and store them in the ```.env``` file.